### DISCUSSION TITLE
Investigate Fido support for login

### DISCUSSION THREAD

**Discussion by teolemon** [openfoodfacts/openfoodfacts-auth#366] [Ideas]
https://stackoverflow.com/questions/26637660/how-do-i-use-fido-u2f-to-allow-users-to-authenticate-with-my-website

**Comment by hangy** [comment]
That StackOverflow answer is a bit outdated - the (proprietary?) Fido U2F login has recently been succeeded by WebAuthn

- https://www.w3.org/2018/04/pressrelease-webauthn-fido2.html.en
- https://www.w3.org/TR/webauthn/

**Comment by hangy** [comment]
Actually somewhat related to openfoodfacts/openfoodfacts-server#542: https://youtu.be/DBBFK7bvEQo?t=29m55s

**Comment by hangy** [comment]
WebAuthn was finalized as a W3C recommendation yesterday: https://www.w3.org/2019/03/pressrelease-webauthn-rec.html
